pkg/api/handlers/projects.go

```diff
@@ -163,10 +168,12 @@ func (h *ProjectsHandler) UpdateProject(req api.Context) error {
 			}
 		}
 
+		thread.Spec.Capabilities = v1.ThreadCapabilities(*project.Capabilities)
```

Comment:
Why do we need `Capabilities` on both the manifest and the spec? Can we get away with just one?

If we only need one, then it should be on the manifest since it can be set via the API.

Also, it seems like it is possible for `project.Capabilities` to be `nil` here. We don't want to panic.

Reply:
My bad, this is extra field and not needed. Removed